LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/snap-after-relayout/snap-to-different-targets-expected.txt

```diff
@@ -1,3 +1,3 @@
 
-PASS Scroller should snap to at least one of the targets if unable to snap toboth after a layout change.
+FAIL Scroller should snap to at least one of the targets if unable to snap toboth after a layout change. assert_true: expected true got false
```

Comment:
Is this FAIL expected?

Reply:
I mention this at the end of the changelog but this is another case of getting the scroll position before we run our relayout code. I will create a follow up bug to fix this test.